pom.xml

```diff
@@ -1462,6 +1485,7 @@
                         <exclude>**/spring.factories</exclude>
                         <exclude>**/spring.provides</exclude>
                         <exclude>**/*.iml</exclude>
+                        <exclude>**/target/**</exclude>
```

Comment:
Only informational: This should already be included due to `.gitignore` (but doesn't hurt here either).

Reply:
.gitignore wasn't working for target, because the rat plugin doesn't ignore what's underneath the directories and git does.